aion/test/channels/question_chronicle_test.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Aion.QuestionChronicleTest do
+  use ExUnit.Case, async: true
+
+  alias Aion.QuestionChronicle
+  @current_time 12345
+  @room_id 1
+
+  setup do
+    QuestionChronicle.update_last_change(@room_id, fn -> @current_time end)
+    assert QuestionChronicle.list_entries == %{@room_id => @current_time}
+    :ok
+  end
+
+  test "should_change is false when the time hasn't passed yet" do
+    time_called = @current_time + QuestionChronicle.question_timeout_micro - 1000
+
+    assert not QuestionChronicle.should_change?(@room_id, fn -> time_called end)
+  end
+
+  test "should_change is false when the time has passed" do
+    time_called = @current_time + QuestionChronicle.question_timeout_micro
+
+    assert QuestionChronicle.should_change?(@room_id, fn -> time_called end)
+  end
+
+end
```

Comment:
This tests are a bit inconsistent (or maybe im not getting something):
1. Why first test returns `:ok` while it has `assert` and the others just have assert?
2. Empty lines on second and third test feel unnecessary.

Reply:
1. It's because it's the `setup` while the others are actual tests.
2. Will fix.